things/api/auth/grpc/endpoint_test.go

```diff
@@ -28,13 +29,15 @@ var (
 )
 
 func TestCanAccessByKey(t *testing.T) {
-	sths, _ := svc.CreateThings(context.Background(), token, thing)
-	oth := sths[0]
-	sths, _ = svc.CreateThings(context.Background(), token, thing)
-	cth := sths[0]
-	schs, _ := svc.CreateChannels(context.Background(), token, channel)
-	sch := schs[0]
-	svc.Connect(context.Background(), token, []string{sch.ID}, []string{cth.ID})
+	ths, err := svc.CreateThings(context.Background(), token, thing, thing)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	th1 := ths[0]
+	th2 := ths[1]
+
+	chs, err := svc.CreateChannels(context.Background(), token, channel)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	ch := chs[0]
+	svc.Connect(context.Background(), token, []string{ch.ID}, []string{th1.ID})
```

Comment:
this returns error, right?

Reply:
Yeah, there was so many unaddressed errors that i obviously missed some.